pkg/erofs/BUILD

```diff
@@ -19,6 +22,7 @@ go_library(
         "//pkg/marshal",
         "//pkg/marshal/primitive",
```

Comment:
This import is no longer used. Can you remove it?

Reply:
Sure, will do. Thanks!